cli/jobs/single-step/gpu_perf/gpu_perf_job.yml

```diff
@@ -0,0 +1,46 @@
+# This runs a couple of tests on a gpu node.
+# It helps diagnosis of health of perf issues.
+
+type: command
+
+code: 
+  local_path: src
+
+command: /bin/bash ./gpu_perf.sh
+
+# to iterate quickly on your environment design, use inline
+environment: file:./environments/azureml/env.yml
```

Comment:
is this "file:" thing needed?

Reply:
I thought it was, removing it worked.

Reply:
Rolled back: if I don't use file:./ the run_job.sh will fail, it looks like a papercut in the way we handle abs/rel file paths there.